art/attacks/iterative_method.py

```diff
@@ -48,99 +46,14 @@ def __init__(self, classifier, norm=np.inf, eps=.3, eps_step=0.1, max_iter=100,
         :type eps: `float`
         :param eps_step: Attack step size (input variation) at each iteration.
         :type eps_step: `float`
-        :param max_iter: The maximum number of iterations.
```

Comment:
I think this parameter should not be removed - can you please double check?

Reply:
On the other hand, I think the parameter `norm` should be removed.

Reply:
I have resolved this with a38b34d8b5cfcbec46cee3b97fb6b9c98ebae283